.github/workflows/rust.yml

```diff
@@ -0,0 +1,29 @@
+name: rust-g
+on: [push]
+jobs:
+  build-windows:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions-rs/cargo@v1
+        with:
+          target: stable-i686-pc-windows-msvc
+          command: build
+          args: --release
+      - uses: actions/upload-artifact@v1
```

Comment:
What happens for commits that aren't tags? Do the binaries still get built? Where do they go?

Reply:
yeah they're still under the actions tab